mlos_bench/mlos_bench/tests/tunables/tunables_assign_test.py

```diff
@@ -141,6 +141,7 @@ def test_tunable_assign_null_to_categorical() -> None:
     categorical_tunable.value = None
     assert categorical_tunable.value is None
     assert categorical_tunable.value != 'None'
+    assert categorical_tunable.categorical_value is None
```

Comment:
Test that we return a non-string for the `categorical_value` when its set to `None`